plaso/parsers/text_plugins/cri.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+"""Text file parser plugin for Container Runtime Interface (CRI) log format.
+
+This is a text-based log format used in kubernetes/GKE.
+
+Also see:
+  https://github.com/kubernetes/design-proposals-archive/blob/main/node/kubelet-cri-logging.md
+"""
+
+import pyparsing
+
+from dfdatetime import golang_time
+
+from plaso.containers import events
+from plaso.lib import errors
+from plaso.parsers import text_parser
+from plaso.parsers.text_plugins import interface
+
+
+class CRIEventData(events.EventData):
+  """CRI log event data.
+
+  Attributes:
+    event_datetime (dfdatetime.Golangtime): the datetime of the log message.
+    message (str): the log message.
+    stream (str): the log stream.  Currently only 'stdout' and 'stderr' are
+        supported.
+    tag (str): the log tag.  Currently only 'P' (partial) and 'F' (full) are
+        supported.
+  """
+  DATA_TYPE = 'cri:container:log:entry'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(CRIEventData, self).__init__(data_type=self.DATA_TYPE)
+    self.event_datetime = None
+    self.message = None
```

Comment:
message is reserved for output field name, please use body or equiv.

Reply:
Done